chroma-manager/tests/integration/installation_and_upgrade/test_update_with_yum.py

```diff
@@ -55,6 +57,11 @@ def test_no_retired_repos(self):
         for repo in retired_repos:
             self.assertFalse(repo in current_repos, "Unexpectedly found repo '%s' in %s" % (repo, current_repos))
 
+    def test_obsolete_chroma_diagnostics(self):
+        """Test that chroma-diagnostics has been obsoleted"""
+        msg = check_output('chroma-diagnostics')
```

Comment:
So, you are running `chroma-diagnostics` here on the test runner, not on any (should be all in fact) of the upgraded nodes.

Reply:
Ok, let me see if I can find how to run a command on the upgraded nodes.